qa/L0_pinned_memory/test.sh

```diff
@@ -148,7 +155,7 @@ for BACKEND in $BACKENDS; do
         RET=1
     fi
 
-    grep "] pinned" ${ENSEMBLE_NAME}.nonpinned.server.log
+    grep "] \"Pinned memory pool is created" ${ENSEMBLE_NAME}.nonpinned.server.log
```

Comment:
Note: This is checking that the following line is **not** found in the log. This was previously "passing" because "pinned" is never found, this is updating the check to be more correct to look for the correct pinned memory message, and assert it's not found when disabled.